app/controllers/admin/application_controller.rb

```diff
@@ -9,9 +9,7 @@ class ApplicationController < Administrate::ApplicationController
     before_action :authenticate_admin
 
     def authenticate_admin
-      if current_user.nil? || current_user.id != 1
-        redirect_to root_path
-      end
+      redirect_to root_path if current_user.nil? || current_user.id != 1
```

Comment:
I've been trying to include parens on most function calls recently (is that a Rubocop option?) because I have nightmares around not knowing Ruby's order of operations. Can we put them on `redirect_to` here?

Reply:
yeap it's ruby-style-guide:
https://github.com/rubocop-hq/ruby-style-guide/blob/master/README.md#if-as-a-modifier

Reply:
I was talking about this one actually: https://github.com/rubocop-hq/ruby-style-guide/blob/master/README.md#method-invocation-parens

But I'm not sure if `redirect_to` counts as part of a Rails DSL.

Reply:
Oh sorry, yes we can use parens with `redirect_to`, but in this case parens - useless.
Let's leave everything as is.